public/components/datasources/components/manage/associated_objects/utils/associated_objects_tab_utils.tsx

```diff
@@ -3,6 +3,19 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { AccelerationIndexType } from '../../../../../../../common/types/data_connections';
+
+export const getAccelerationType = (type: AccelerationIndexType) => {
```

Comment:
Can re-use this: https://github.com/opensearch-project/dashboards-observability/blob/8874c8c6ff778221e633fc8c52356a78f878c22c/common/constants/data_sources.ts#L41-L45

Reply:
Something like this?
```
ACCELERATION_INDEX_TYPES.find((accelType) => accelType.value === acceleration.type)!.label
```